dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/export/OperationsParamsValidatorTest.java

```diff
@@ -131,34 +125,24 @@ void shouldFailWhenOuModeCaptureAndUserHasNoOrgUnitsAssigned() {
   void shouldFailWhenOuModeRequiresUserScopeOrgUnitAndUserHasNoOrgUnitsAssigned(
       OrganisationUnitSelectionMode orgUnitMode) {
 
-    try (MockedStatic<CurrentUserUtil> userUtilMockedStatic = mockStatic(CurrentUserUtil.class)) {
-      userUtilMockedStatic
-          .when(CurrentUserUtil::getCurrentUserDetails)
-          .thenReturn(UserDetails.fromUser(new User()));
-      Exception exception =
-          Assertions.assertThrows(
-              BadRequestException.class, () -> validateOrgUnitMode(orgUnitMode, program));
-
-      assertEquals(
-          "User needs to be assigned either search or data capture org units",
-          exception.getMessage());
-    }
+    Exception exception =
+        Assertions.assertThrows(
+            BadRequestException.class, () -> validateOrgUnitMode(orgUnitMode, program, user));
+
+    assertEquals(
+        "User needs to be assigned either search or data capture org units",
+        exception.getMessage());
   }
 
   @Test
   void shouldFailWhenOuModeAllAndNotSuperuser() {
-    try (MockedStatic<CurrentUserUtil> userUtilMockedStatic = mockStatic(CurrentUserUtil.class)) {
-      userUtilMockedStatic
-          .when(CurrentUserUtil::getCurrentUserDetails)
-          .thenReturn(UserDetails.fromUser(new User()));
-      Exception exception =
-          Assertions.assertThrows(
-              BadRequestException.class, () -> validateOrgUnitMode(ALL, program));
-
-      assertEquals(
-          "Current user is not authorized to query across all organisation units",
-          exception.getMessage());
-    }
+    Exception exception =
+        Assertions.assertThrows(
+            BadRequestException.class, () -> validateOrgUnitMode(ALL, program, user));
+
+    assertEquals(
+        "Current user is not authorized to query across all organisation units",
```

Comment:
```suggestion
        "User is not authorized to query across all organisation units",
```

looks like most other messages just say user

Reply:
Done